src/app/Antelope/index.ts

```diff
@@ -0,0 +1,15 @@
+import antelopeImage from './antelope.png';
+import { Animal } from '../Animal';
+
+const ANTELOPE_STRENGTH = 2;
```

Comment:
In general, I'm not using all-uppercase for constants, I don't think it looks too good. But I understand that's a somewhat common approach :+1: